hepdata_converter/writers/yoda_writer.py

```diff
@@ -120,10 +122,8 @@ def _write_table(self, data_out, table):
                     if qualifier['name'] == 'Custom Rivet identifier':
                         rivet_identifier = qualifier['value']
             rivet_path = '/REF/' + self.rivet_analysis_name + '/' + rivet_identifier
-            graph.title = table_doi  # use for YODA 1.7.7
-            graph.path = rivet_path  # use for YODA 1.7.7
-#            graph.setTitle(table_doi)  # use for YODA 1.8.1
-#            graph.setPath(rivet_path)  # use for YODA 1.8.1
+            graph.setTitle(table_doi)  # use for YODA 1.8.1
+            graph.setPath(rivet_path)  # use for YODA 1.8.1
```

Comment:
Change comment to `# use for YODA > 1.8.0`.

Reply:
Should I just delete the comments now we don't have two versions of the code?

Reply:
Yes, better just to delete the comment, since we won't downgrade YODA in future.